docs/v7_migration.rst

```diff
@@ -224,6 +208,42 @@ errors:
 - ``TransactionIndexingInProgress``
 
 
+End of Support and Feature Removals
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Python 3.7 Support Dropped
+``````````````````````````
+
+Python 3.7 support has been dropped in favor of Python 3.8+. Python 3.7 is no longer
+supported by the Python core team, and we want to focus our efforts on supporting
+the latest versions of Python.
+
+
+EthPM Module Removed
+````````````````````
+
+The EthPM module has been removed from the library. It was not widely used and has not
+been functional since around October 2022. It was deprecated in ``v6`` and has been
+completely removed in ``v7``.
+
+
+Geth Miner Namespace Removed
+````````````````````````````
+
+The ``geth.miner`` namespace, deprecated in ``v6``, has been removed in ``v7``.
```

Comment:
I may be looking at the wrong thing, but I don't think we deprecated miner and personal? I think we may have just removed them?

Reply:
Welp, I s'pose those should be backported 😅